### PR TITLE
Fix alpha-bugs and update User Guide

### LINKED ISSUE #132: Sort/List Command: No way to reverse alphabetical sorting
### Description
Upon executing the `sort` command, there is no way to reverse this sorting and return to alphabetical sorting (aside from exiting and reopening the application or executing other commands - BUT doesn't work for `list`). 

### Suggested improvement
- Maybe an `unsort` command to return to alphabetical sorting
- Or if `sort` is executed again while contacts are sorted by appointments it will sort alphabetically
- Or `sort` can take in arguments e.g. `sort appointments` will sort by appointment date and `sort alpha` will sort alphabetically
- Or `list` will automatically list in alphabetical order so user can execute `list` after `sort` if they wish to return to alphabetical sorting

### PR DESCRIPTION
**Pull request by Bonzzz3**
### **Description**
Fix alpha-bugs and update User Guide

### **Key Changes**
- Fix confirmation window for clear showing incorrect title
- Fix list command not sorting persons alphabetically
- Update command details for clear, delete and list
- Update screenshots of delete, clear and help window

Fixes #132 #137

### CODE REVIEW

**Review by LynetteLeeSiYing — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -109,16 +109,22 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+> [!WARNING]
+> After confirmation, any cleared data or contact cannot be retrieved. This action is IRREVERSIBLE.
+
+* Recommended practice: Always have a copy of current data (refer to the export feature). You can use import feature if you have lost your data after clearing it unintentionally.
```
Comment (myrilla12) [inline docs/UserGuide.md:115]: Can include hyperlink to `Exporting data` section and `Importing data` in a similar format to how `import` feature section has hyperlinks

**Review by myrilla12 — APPROVED** [review]
Just one tiny suggestion on hyperlinks, can also do in a future PR. LGTM!

**Review by ritvi12 — APPROVED** [review]
LGTM!

**Pull request merged**